BE/LegalEntities/FormalBusinessOrganizations.rdf

```diff
@@ -117,7 +121,7 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>non-governmental organization</rdfs:label>
-		<skos:definition>a not for profit organization that is a citizen-based group that functions independently of government</skos:definition>
+		<skos:definition>not-for-profit organization that is a citizen-based group that functions independently of government</skos:definition>
```

Comment:
"citizen-based" is meaningless

Reply:
Removed, as well as a related explanatory note.